Flowboard >> page isn't refreshed anymore when settings are opened

MigDinny opened the pull request:
Now the refreshing feature will only refresh the flowboard if settings aren't opened.

Nothing else to add.

Fixes #831

teryhill commented:
looks good @MigDinny . I did notice that the first and second option are the same. Must have missed that earlier. Also if you change the provider to All then change a setting it resets the previous screen settings.

![image](https://user-images.githubusercontent.com/4655366/33690481-abe07916-dab1-11e7-9566-d0001a568b8e.png)

teryhill commented:
![image](https://user-images.githubusercontent.com/4655366/33690777-2ade5322-dab3-11e7-8f28-293ee1cafb17.png)

teryhill commented:
@MigDinny  go to users and add another provider and you will see the issue I am speaking or change the date range in the above screen shot

MigDinny commented:
I added a new user but it doesn't even show up on the Provider selection.

Changed date range and didn't notice anything. What are you exactly talking about? Can you explain?

teryhill commented:
Did you check the provider and calendar check box

![capture](https://user-images.githubusercontent.com/4655366/33731529-69dc6bc4-db51-11e7-829c-6fe096364286.PNG)

MigDinny commented:
I did.

teryhill commented:
What does your flowboard say under provider?

MigDinny commented:
![image](https://user-images.githubusercontent.com/6632800/33735336-eed36e56-db86-11e7-8beb-778953de93e3.png)

teryhill commented:
@migdinny post a screen shot of your users

MigDinny commented:
@teryhill look again

teryhill commented:
Open up the user test and do a screen shot @MigDinny

MigDinny commented:
![image](https://user-images.githubusercontent.com/6632800/33736129-67031a00-db89-11e7-8fbd-72f53257b72e.png)


can we talk on chat? it's easier and faster

MigDinny commented:
Duplicated option removed.

teryhill commented on interface/patient_tracker/patient_tracker.php:
```diff
@@ -205,46 +211,21 @@ function openNewTopWindow(newpid,newencounterid) {
-        <label><input type='checkbox' name='form_new_window' value='1' <?php echo $new_window_checked; ?> /><?= xlt('Open Patient in New Window'); ?></label>
-        </div>
         <div class="checkbox">
         <label><input type="checkbox" id="ptkr_pt_list_new_window" name="ptkr_pt_list_new_window" value="1" <?php if($GLOBALS['ptkr_pt_list_new_window']=='1') echo "checked"; ?>><?php echo xlt("Open Demographics in New Window from Patient Flow Board"); ?></label>
```
Echo new window checked here

MigDinny replied:
what do you mean?

teryhill commented:
This is good to go @aethelwulffe @tmccormi @Trodrige  more eyes

teryhill commented:
@aethelwulffe , @tmccormi Can one of you look at this ?

teryhill commented:
@tmccormi  and @aethelwulffe  can you guys look at these PR's
